apps/matrix/main.py

```diff
@@ -230,7 +273,11 @@ def _on_message(self, room, event):
 				prefix = ""
 				if event.get('sender', None) == self.client.get_user().user_id or event.get("sender", None) in self.config["your_other_usernames"]:
 					# Prefix own messages with a '*'
-					prefix = "* "
+					prefix = "[*] "
```

Comment:
Are you sure you want to add two more characters? We don't have too many of them on the screen =)

Reply:
Was wondering about this as well, I'll change it back to one =)